dandiapi/api/models/asset.py

```diff
@@ -147,4 +147,6 @@ def get_path(cls, path_prefix: str, qs: List[str]) -> Set:
 
     @classmethod
     def total_size(cls):
-        return cls.objects.aggregate(size=models.Sum('blob__size'))['size'] or 0
+        return (
+            cls.objects.filter(versions__gt=0).aggregate(size=models.Sum('blob__size'))['size'] or 0
```

Comment:
have you looked at #298 ?  this would not be sufficient AFAIK since would double count the same blobs across multiple versions/instances of dandisets and I believe we shouldn't do that

Reply:
Good point, I have fixed this